packages/jest-serializer-merge-styles/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es5",
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=27944&commitId=76bbbe8ce22f85895971c27e4cd127f6905827ee&env=prod&clientType=FLUENTUI)


  <details><summary><b>(Dev-Only) Suggestions</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over the X button on a wide suggestion element.chromium.png|11211| Changed|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.default.chromium.png|11375| Changed|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over a wide suggestion element.chromium.png|11277| Changed|

</details>


  <details><summary><b>ActivityItem</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.CompactPersonas.default.chromium.png|3479| Changed|
|ActivityItem.Compact.default.chromium.png|2596| Changed|
|ActivityItem.Personas - RTL.default.chromium.png|11053| Changed|
|ActivityItem.Root.default.chromium.png|4526| Changed|
|ActivityItem.Root - RTL.default.chromium.png|4495| Changed|
|ActivityItem.Personas.default.chromium.png|11347| Changed|

</details>


  <details><summary><b>Breadcrumb</b> 33 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb.Button - RTL.overflowButtonFocus.chromium.png|69565| Changed|
|Breadcrumb.Button.click.chromium.png|69724| Changed|
|Breadcrumb.Button - RTL.itemLinkFocus.chromium.png|69556| Changed|
|Breadcrumb.Button - RTL.default.chromium.png|69563| Changed|
|Breadcrumb.Button.default.chromium.png|69256| Changed|
|Breadcrumb.Button.hover.chromium.png|69693| Changed|
|Breadcrumb.Button - RTL.shortTitleHover.chromium.png|69414| Changed|
|Breadcrumb.Button.overflowButtonFocus.chromium.png|69429| Changed|
|Breadcrumb.Button - RTL.longTitleHover.chromium.png|69378| Changed|
|Breadcrumb.Button.itemLinkFocus.chromium.png|69670| Changed|
|Breadcrumb.Button - RTL.hover.chromium.png|69715| Changed|
|Breadcrumb.Button - RTL.click.chromium.png|68982| Changed|
|Breadcrumb.Button.longTitleHover.chromium.png|69439| Changed|
|Breadcrumb.Hovering items.non-actionable hover.chromium.png|57532| Changed|
|Breadcrumb.Hovering items.actionable hover.chromium.png|57676| Changed|
|Breadcrumb.Hovering items.non-actionable overflow hover.chromium.png|57991| Changed|
|Breadcrumb.Button.shortTitleHover.chromium.png|69070| Changed|
|Breadcrumb.Hovering items.actionable overflow hover.chromium.png|58120| Changed|
|Breadcrumb.Root - RTL.itemLinkFocus.chromium.png|81453| Changed|
|Breadcrumb.Root - RTL.longTitleHover.chromium.png|80638| Changed|
|Breadcrumb.Root.click.chromium.png|81420| Changed|
|Breadcrumb.Root - RTL.default.chromium.png|81185| Changed|
|Breadcrumb.Root.default.chromium.png|80941| Changed|
|Breadcrumb.Root.itemLinkFocus.chromium.png|81956| Changed|
|Breadcrumb.Root - RTL.click.chromium.png|80680| Changed|
|Breadcrumb.Root - RTL.shortTitleHover.chromium.png|81359| Changed|
|Breadcrumb.Root - RTL.overflowButtonFocus.chromium.png|81194| Changed|
|Breadcrumb.Root - RTL.hover.chromium.png|81770| Changed|
|Breadcrumb.Root.shortTitleHover.chromium.png|80704| Changed|
|Breadcrumb.Root.longTitleHover.chromium.png|81202| Changed|
|Breadcrumb.Root.hover.chromium.png|81922| Changed|
|Breadcrumb.Root.overflowButtonFocus.chromium.png|81113| Changed|
|Breadcrumb.Hovering items.chromium.png|1763| Changed|

</details>


  <details><summary><b>Button (compat)</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button (compat).Disabled.default.chromium.png|3344| Changed|
|Button (compat).Checked.hover.chromium.png|3484| Changed|
|Button (compat).Disabled.hover.chromium.png|3344| Changed|
|Button (compat).Disabled.pressed.chromium.png|3344| Changed|
|Button (compat).Icon Only.default.chromium.png|3451| Changed|
|Button (compat).Icon Only.hover.chromium.png|3407| Changed|
|Button (compat).No Icon.default.chromium.png|2535| Changed|
|Button (compat).Checked.default.chromium.png|3456| Changed|
|Button (compat).Checked.pressed.chromium.png|3457| Changed|
|Button (compat).No Icon.hover.chromium.png|2554| Changed|
|Button (compat).Primary Anchor.default.chromium.png|3023| Changed|
|Button (compat).No Icon.pressed.chromium.png|2507| Changed|
|Button (compat).Icon Only.pressed.chromium.png|3346| Changed|
|Button (compat).Primary Checked.default.chromium.png|3736| Changed|
|Button (compat).Primary Checked.pressed.chromium.png|3768| Changed|
|Button (compat).Primary Anchor.pressed.chromium.png|3022| Changed|
|Button (compat).Primary Anchor.hover.chromium.png|3023| Changed|
|Button (compat).Primary Disabled.pressed.chromium.png|3336| Changed|
|Button (compat).Primary.hover.chromium.png|3641| Changed|
|Button (compat).Primary Disabled.default.chromium.png|3336| Changed|
|Button (compat).Primary.default.chromium.png|3673| Changed|
|Button (compat).Primary.pressed.chromium.png|3774| Changed|
|Button (compat).Root - RTL.pressed.chromium.png|3477| Changed|
|Button (compat).Root.default.chromium.png|3536| Changed|
|Button (compat).Root.hover.chromium.png|3528| Changed|
|Button (compat).Primary Checked.hover.chromium.png|3690| Changed|
|Button (compat).Root.pressed.chromium.png|3447| Changed|
|Button (compat).Root - RTL.hover.chromium.png|3554| Changed|
|Button (compat).Root - RTL.default.chromium.png|3562| Changed|
|Button (compat).Primary Disabled.hover.chromium.png|3336| Changed|

</details>


  <details><summary><b>Button Action (compat)</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Action (compat).Checked.default.chromium.png|3231| Changed|
|Button Action (compat).Checked.pressed.chromium.png|3170| Changed|
|Button Action (compat).Icon Only.hover.chromium.png|1626| Changed|
|Button Action (compat).Icon Only.pressed.chromium.png|1585| Changed|
|Button Action (compat).Disabled.pressed.chromium.png|3035| Changed|
|Button Action (compat).No Icon.default.chromium.png|2029| Changed|
|Button Action (compat).Checked.hover.chromium.png|3284| Changed|
|Button Action (compat).Icon Only.default.chromium.png|1604| Changed|
|Button Action (compat).Disabled.default.chromium.png|3035| Changed|
|Button Action (compat).Root - RTL.hover.chromium.png|3251| Changed|
|Button Action (compat).Disabled.hover.chromium.png|3035| Changed|
|Button Action (compat).No Icon.hover.chromium.png|2061| Changed|
|Button Action (compat).Root - RTL.default.chromium.png|3214| Changed|
|Button Action (compat).No Icon.pressed.chromium.png|1994| Changed|
|Button Action (compat).Root.default.chromium.png|3209| Changed|
|Button Action (compat).Root - RTL.pressed.chromium.png|3177| Changed|
|Button Action (compat).Root.hover.chromium.png|3248| Changed|
|Button Action (compat).Root.pressed.chromium.png|3170| Changed|

</details>


  <details><summary><b>Button Command (compat)</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Command (compat).Disabled.default.chromium.png|6341| Changed|
|Button Command (compat).Root - RTL.default.chromium.png|6607| Changed|
|Button Command (compat).Checked.default.chromium.png|6331| Changed|
|Button Command (compat).Disabled.hover.chromium.png|6341| Changed|
|Button Command (compat).Disabled.open.chromium.png|6341| Changed|
|Button Command (compat).Checked.hover.chromium.png|6321| Changed|
|Button Command (compat).Root.hover.chromium.png|6587| Changed|
|Button Command (compat).Checked.pressed.chromium.png|6564| Changed|
|Button Command (compat).Root.pressed.chromium.png|6308| Changed|
|Button Command (compat).Root - RTL.open.chromium.png|6615| Changed|
|Button Command (compat).Root - RTL.hover.chromium.png|6564| Changed|
|Button Command (compat).Root - RTL.pressed.chromium.png|6280| Changed|
|Button Command (compat).Checked.open.chromium.png|6836| Changed|
|Button Command (compat).Root.default.chromium.png|6633| Changed|
|Button Command (compat).Root.open.chromium.png|6836| Changed|
|Button Command (compat).Disabled.pressed.chromium.png|6341| Changed|

</details>


  <details><summary><b>Button Compound (compat)</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Compound (compat).Disabled.default.chromium.png|25776| Changed|
|Button Compound (compat).No Icon.hover.chromium.png|25222| Changed|
|Button Compound (compat).Disabled.pressed.chromium.png|25776| Changed|
|Button Compound (compat).Checked.hover.chromium.png|24791| Changed|
|Button Compound (compat).Primary Disabled.hover.chromium.png|25921| Changed|
|Button Compound (compat).Checked.default.chromium.png|25001| Changed|
|Button Compound (compat).No Icon.pressed.chromium.png|24901| Changed|
|Button Compound (compat).Checked.pressed.chromium.png|24985| Changed|
|Button Compound (compat).Primary Checked.pressed.chromium.png|22778| Changed|
|Button Compound (compat).Root - RTL.default.chromium.png|26261| Changed|
|Button Compound (compat).Primary Disabled.default.chromium.png|25921| Changed|
|Button Compound (compat).Root.default.chromium.png|26261| Changed|
|Button Compound (compat).Root.pressed.chromium.png|25073| Changed|
|Button Compound (compat).Disabled.hover.chromium.png|25776| Changed|
|Button Compound (compat).Primary Checked.hover.chromium.png|22031| Changed|
|Button Compound (compat).No Icon.default.chromium.png|26110| Changed|
|Button Compound (compat).Primary.default.chromium.png|22862| Changed|
|Button Compound (compat).Primary.hover.chromium.png|22363| Changed|
|Button Compound (compat).Primary Disabled.pressed.chromium.png|25921| Changed|
|Button Compound (compat).Primary Checked.default.chromium.png|22489| Changed|
|Button Compound (compat).Primary.pressed.chromium.png|22776| Changed|
|Button Compound (compat).Root - RTL.hover.chromium.png|25371| Changed|
|Button Compound (compat).Root - RTL.pressed.chromium.png|25072| Changed|
|Button Compound (compat).Root.hover.chromium.png|25371| Changed|

</details>


  <details><summary><b>Button Special Scenarios (compat)</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Special Scenarios (compat).primary with placeholder.default.chromium.png|5976| Changed|
|Button Special Scenarios (compat).no flex shrink.default.chromium.png|4331| Changed|

</details>


  <details><summary><b>Button Split (compat)</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Split (compat).Checked.default.chromium.png|11230| Changed|
|Button Split (compat).Checked.hover main.chromium.png|11297| Changed|
|Button Split (compat).Command Split.hover split.chromium.png|6281| Changed|
|Button Split (compat).Default with Primary Action Disabled.hover split.chromium.png|10906| Changed|
|Button Split (compat).Checked.hover split.chromium.png|11236| Changed|
|Button Split (compat).Command Split.hover main.chromium.png|6294| Changed|
|Button Split (compat).Command Split.default.chromium.png|6271| Changed|
|Button Split (compat).Primary Checked.hover main.chromium.png|12019| Changed|
|Button Split (compat).Primary Disabled.hover main.chromium.png|10711| Changed|
|Button Split (compat).Primary Disabled.default.chromium.png|10711| Changed|
|Button Split (compat).Primary with Primary Action Disabled.hover main.chromium.png|10231| Changed|
|Button Split (compat).Primary.hover split.chromium.png|11868| Changed|
|Button Split (compat).Primary.hover main.chromium.png|11863| Changed|
|Button Split (compat).Default with Primary Action Disabled.default.chromium.png|10952| Changed|
|Button Split (compat).Root - RTL.default.chromium.png|11389| Changed|
|Button Split (compat).Root.default.chromium.png|11380| Changed|
|Button Split (compat).Default with Primary Action Disabled.hover main.chromium.png|10952| Changed|
|Button Split (compat).Root.hover main.chromium.png|11430| Changed|
|Button Split (compat).Root - RTL.hover main.chromium.png|11401| Changed|
|Button Split (compat).Disabled.hover main.chromium.png|10919| Changed|
|Button Split (compat).Primary Checked.default.chromium.png|12122| Changed|
|Button Split (compat).Disabled.hover split.chromium.png|10919| Changed|
|Button Split (compat).Primary Disabled.hover split.chromium.png|10711| Changed|
|Button Split (compat).Primary.default.chromium.png|11895| Changed|
|Button Split (compat).Primary with Primary Action Disabled.hover split.chromium.png|10240| Changed|
|Button Split (compat).Primary with Primary Action Disabled.default.chromium.png|10184| Changed|
|Button Split (compat).Primary Checked.hover split.chromium.png|12102| Changed|
|Button Split (compat).Root - RTL.hover split.chromium.png|11381| Changed|
|Button Split (compat).Disabled.default.chromium.png|10919| Changed|
|Button Split (compat).Root.hover split.chromium.png|11328| Changed|

</details>


  <details><summary><b>Calendar</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar.Root - RTL.default.chromium.png|49676| Changed|
|Calendar.Root.default.chromium.png|49662| Changed|

</details>


  <details><summary><b>Calendar - No Month Option</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar - No Month Option.Show Month selection type.default.chromium.png|49622| Changed|
|Calendar - No Month Option.Show Week selection type.default.chromium.png|49693| Changed|
|Calendar - No Month Option.Show Month as Overlay and no Go To Today.default.chromium.png|14573| Changed|
|Calendar - No Month Option.Show Work Week selection type.default.chromium.png|49693| Changed|

</details>


  <details><summary><b>Callout</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left bottom edge.default.chromium.png|5487| Changed|
|Callout.Left top edge.default.chromium.png|5477| Changed|
|Callout.No beak.default.chromium.png|5520| Changed|
|Callout.Bottom left edge.default.chromium.png|5526| Changed|
|Callout.Gap space 25.default.chromium.png|5528| Changed|
|Callout.Bottom right edge.default.chromium.png|5527| Changed|
|Callout.Bottom left edge - RTL.default.chromium.png|5544| Changed|
|Callout.Bottom right edge - RTL.default.chromium.png|5544| Changed|
|Callout.Beak 25.default.chromium.png|5529| Changed|
|Callout.Left center.default.chromium.png|5427| Changed|
|Callout.Bottom center.default.chromium.png|5527| Changed|
|Callout.Bottom auto edge.default.chromium.png|5526| Changed|
|Callout.Right center.default.chromium.png|5391| Changed|
|Callout.Right top edge.default.chromium.png|5527| Changed|
|Callout.No callout width specified.default.chromium.png|5601| Changed|
|Callout.Right bottom edge.default.chromium.png|5515| Changed|
|Callout.Rendering callout attached to a rectangle.default.chromium.png|5528| Changed|
|Callout.Root.default.chromium.png|5526| Changed|
|Callout.Top auto edge.default.chromium.png|5474| Changed|
|Callout.Top center.default.chromium.png|5473| Changed|
|Callout.Top left edge.default.chromium.png|5474| Changed|
|Callout.Top right edge.default.chromium.png|5473| Changed|

</details>


  <details><summary><b>Checkbox</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checked disabled.default.chromium.png|7409| Changed|
|Checkbox.Controlled Indeterminate.default.chromium.png|9325| Changed|
|Checkbox.Checked.default.chromium.png|5819| Changed|
|Checkbox.End - RTL.default.chromium.png|4827| Changed|
|Checkbox.Unchecked disabled.default.chromium.png|7877| Changed|
|Checkbox.Unchecked - RTL.default.chromium.png|6238| Changed|
|Checkbox.Custom render Checkbox.default.chromium.png|9373| Changed|
|Checkbox.End.default.chromium.png|4817| Changed|
|Checkbox.Uncontrolled Indeterminate.default.chromium.png|9848| Changed|
|Checkbox.Unchecked.default.chromium.png|6263| Changed|
|Checkbox.Uncontrolled Indeterminate disabled.default.chromium.png|11324| Changed|
|Checkbox.Controlled Indeterminate disabled.default.chromium.png|10820| Changed|
|Checkbox.Multi-line Checkbox.default.chromium.png|83891| Changed|

</details>


  <details><summary><b>Checkbox Indeterminate</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.clicked.chromium.png|15291| Changed|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.hover.chromium.png|15280| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.default.chromium.png|9848| Changed|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.default.chromium.png|15190| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.hover.chromium.png|9909| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.default.chromium.png|13252| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.clicked.chromium.png|13380| Changed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.default.chromium.png|13154| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.clicked.chromium.png|9864| Changed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.hover.chromium.png|13233| Changed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.clicked.chromium.png|13243| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.hover.chromium.png|13333| Changed|

</details>


  <details><summary><b>ChoiceGroup</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ChoiceGroup.Required.default.chromium.png|6960| Changed|
|ChoiceGroup.Required.selected.chromium.png|7044| Changed|
|ChoiceGroup.Required.hover unselected.chromium.png|7006| Changed|
|ChoiceGroup.Root.hover selected.chromium.png|7036| Changed|
|ChoiceGroup.Root.hover unselected.chromium.png|6998| Changed|
|ChoiceGroup.Root.selected.chromium.png|7036| Changed|
|ChoiceGroup.With default size images.selected.chromium.png|24229| Changed|
|ChoiceGroup.With default size images.hover unselected.chromium.png|24084| Changed|
|ChoiceGroup.With icons - RTL.default.chromium.png|35486| Changed|
|ChoiceGroup.With icons - RTL.selected.chromium.png|35191| Changed|
|ChoiceGroup.With default size images.hover selected.chromium.png|24229| Changed|
|ChoiceGroup.With icons - RTL.hover unselected.chromium.png|35179| Changed|
|ChoiceGroup.With default size images.default.chromium.png|23912| Changed|
|ChoiceGroup.With icons.default.chromium.png|36421| Changed|
|ChoiceGroup.Required.hover selected.chromium.png|7044| Changed|
|ChoiceGroup.Root.default.chromium.png|6952| Changed|
|ChoiceGroup.With icons - RTL.hover selected.chromium.png|35191| Changed|
|ChoiceGroup.With icons.selected.chromium.png|36739| Changed|
|ChoiceGroup.With icons.hover selected.chromium.png|36739| Changed|
|ChoiceGroup.With icons.hover unselected.chromium.png|36694| Changed|
|ChoiceGroup.With large size images.hover unselected.chromium.png|97664| Changed|
|ChoiceGroup.With large size images.hover selected.chromium.png|98893| Changed|
|ChoiceGroup.With large size images.default.chromium.png|97694| Changed|
|ChoiceGroup.With large size images.selected.chromium.png|98893| Changed|

</details>


  <details><summary><b>Coachmark</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Rendering Coachmark attached to a rectangle.default.chromium.png|1839389| Changed|
|Coachmark.Collapsed.default.chromium.png|1763656| Changed|

</details>


  <details><summary><b>ColorPicker</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ColorPicker.Root.default.chromium.png|107493| Changed|
|ColorPicker.Blue.default.chromium.png|107743| Changed|
|ColorPicker.Root - RTL.default.chromium.png|106799| Changed|

</details>


  <details><summary><b>ComboBox</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ComboBox.Disabled.hover.chromium.png|7848| Changed|
|ComboBox.Disabled.default.chromium.png|7789| Changed|
|ComboBox.Error.default.chromium.png|9621| Changed|
|ComboBox.Error.hover.chromium.png|9743| Changed|
|ComboBox.Placeholder.default.chromium.png|8135| Changed|
|ComboBox.Placeholder.hover.chromium.png|8359| Changed|
|ComboBox.Root - RTL.default.chromium.png|8223| Changed|
|ComboBox.Root.default.chromium.png|8228| Changed|
|ComboBox.Root - RTL.hover.chromium.png|8392| Changed|
|ComboBox.Root.hover.chromium.png|8397| Changed|
|ComboBox.Styled.default.chromium.png|8197| Changed|
|ComboBox.Styled.hover.chromium.png|8367| Changed|
|ComboBox.Root - RTL.click.chromium.png|1826418| Changed|
|ComboBox.Styled.click.chromium.png|1828920| Changed|
|ComboBox.Disabled.click.chromium.png|1095| Changed|
|ComboBox.Placeholder.click.chromium.png|1828196| Changed|
|ComboBox.Error.click.chromium.png|1828123| Changed|
|ComboBox.Root.click.chromium.png|1828366| Changed|

</details>


  <details><summary><b>CommandBar</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CommandBar.Icons only.default.chromium.png|65987| Changed|
|CommandBar.Icons only.click.chromium.png|65338| Changed|
|CommandBar.Icons only.hover.chromium.png|65952| Changed|
|CommandBar.Root - RTL.click.chromium.png|122785| Changed|
|CommandBar.Root - RTL.hover.chromium.png|123666| Changed|
|CommandBar.Root.hover.chromium.png|123808| Changed|
|CommandBar.Root.click.chromium.png|123683| Changed|
|CommandBar.Root.default.chromium.png|123785| Changed|
|CommandBar.Root - RTL.default.chromium.png|123809| Changed|
|CommandBar.Text only.default.chromium.png|90152| Changed|
|CommandBar.Text only.hover.chromium.png|90221| Changed|
|CommandBar.Text only.click.chromium.png|90147| Changed|

</details>


  <details><summary><b>ContextualMenu</b> 29 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With headers.hover.chromium.png|1824098| Changed|
|ContextualMenu.Root.hover.chromium.png|1824849| Changed|
|ContextualMenu.With headers.default.chromium.png|1824098| Changed|
|ContextualMenu.Root.default.chromium.png|1824744| Changed|
|ContextualMenu.With headers.click.chromium.png|1823949| Changed|
|ContextualMenu.With icons.hover.chromium.png|1832026| Changed|
|ContextualMenu.Root.click.chromium.png|1824849| Changed|
|ContextualMenu.With icons.default.chromium.png|1832174| Changed|
|ContextualMenu.With icons.click.chromium.png|1832026| Changed|
|ContextualMenu.With secondaryText - RTL.default.chromium.png|1827226| Changed|
|ContextualMenu.With secondaryText - RTL.click.chromium.png|1827125| Changed|
|ContextualMenu.With secondaryText.click.chromium.png|1827124| Changed|
|ContextualMenu.With secondaryText - RTL.hover.chromium.png|1827125| Changed|
|ContextualMenu.With secondaryText.default.chromium.png|1827224| Changed|
|ContextualMenu.With split button submenu.click.chromium.png|1841180| Changed|
|ContextualMenu.With secondaryText.hover.chromium.png|1827124| Changed|
|ContextualMenu.With split button submenu.default.chromium.png|1841531| Changed|
|ContextualMenu.With submenu - RTL.click.chromium.png|1837194| Changed|
|ContextualMenu.With split button submenu.hover.chromium.png|1841277| Changed|
|ContextualMenu.With submenu - RTL.hover.chromium.png|1837194| Changed|
|ContextualMenu.With submenu.default.chromium.png|1840099| Changed|
|ContextualMenu.With submenu.click.chromium.png|1837316| Changed|
|ContextualMenu.With submenu - RTL.default.chromium.png|1839980| Changed|
|ContextualMenu.With submenu.hover.chromium.png|1837316| Changed|
|ContextualMenu.With submenus with hrefs.item2 hovered.chromium.png|1838779| Changed|
|ContextualMenu.With submenus with hrefs.chromium.png|1083| Changed|
|ContextualMenu.With submenus with hrefs.item1 hovered.chromium.png|1838786| Changed|
|ContextualMenu.With submenus with hrefs.parent hovered.chromium.png|1839776| Changed|
|ContextualMenu.With submenus with hrefs.menu opened.chromium.png|1840516| Changed|

</details>


  <details><summary><b>DatePicker</b> 35 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker.Allow text input.default.chromium.png|5562| Changed|
|DatePicker.Allow text input.hover datepicker.chromium.png|5569| Changed|
|DatePicker.Placeholder.default.chromium.png|1210| Changed|
|DatePicker.Placeholder.hover datepicker.chromium.png|1326| Changed|
|DatePicker.Required.default.chromium.png|1217| Changed|
|DatePicker.Required.hover datepicker.chromium.png|1333| Changed|
|DatePicker.Root - RTL.default.chromium.png|1213| Changed|
|DatePicker.Root - RTL.hover datepicker.chromium.png|1328| Changed|
|DatePicker.Root.default.chromium.png|1210| Changed|
|DatePicker.Root.hover datepicker.chromium.png|1326| Changed|
|DatePicker.Underlined and Required.default.chromium.png|1330| Changed|
|DatePicker.Underlined and Required.hover datepicker.chromium.png|1383| Changed|
|DatePicker.Underlined.hover datepicker.chromium.png|1376| Changed|
|DatePicker.Underlined.default.chromium.png|1323| Changed|
|DatePicker.Allow text input.hover day.chromium.png|1759373| Changed|
|DatePicker.Allow text input.click.chromium.png|1759404| Changed|
|DatePicker.Placeholder.click.chromium.png|3355| Changed|
|DatePicker.Allow text input.hover month.chromium.png|1759271| Changed|
|DatePicker.Required.click.chromium.png|3362| Changed|
|DatePicker.Root.click.chromium.png|3355| Changed|
|DatePicker.Root - RTL.click.chromium.png|3358| Changed|
|DatePicker.Underlined and Required.click.chromium.png|3093| Changed|
|DatePicker.Underlined.click.chromium.png|3086| Changed|
|DatePicker.Placeholder.hover day.chromium.png|0| Removed|
|DatePicker.Placeholder.hover month.chromium.png|0| Removed|
|DatePicker.Required.hover day.chromium.png|0| Removed|
|DatePicker.Required.hover month.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover day.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover month.chromium.png|0| Removed|
|DatePicker.Root.hover day.chromium.png|0| Removed|
|DatePicker.Root.hover month.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover day.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover month.chromium.png|0| Removed|
|DatePicker.Underlined.hover day.chromium.png|0| Removed|
|DatePicker.Underlined.hover month.chromium.png|0| Removed|

</details>


  <details><summary><b>DatePicker - Disabled</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker - Disabled.With Label.default.chromium.png|9264| Changed|
|DatePicker - Disabled.With Label.hover datepicker.chromium.png|9264| Changed|
|DatePicker - Disabled.Without Label.default.chromium.png|5652| Changed|
|DatePicker - Disabled.Without Label.hover datepicker.chromium.png|5652| Changed|
|DatePicker - Disabled.Without Value.default.chromium.png|9076| Changed|
|DatePicker - Disabled.Without Value.hover datepicker.chromium.png|9076| Changed|
|DatePicker - Disabled.With Label.click.chromium.png|1068| Changed|
|DatePicker - Disabled.Without Label.click.chromium.png|582| Changed|
|DatePicker - Disabled.Without Value.click.chromium.png|846| Changed|

</details>


  <details><summary><b>DatePicker - No Month Option</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.default.chromium.png|1210| Changed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover datepicker.chromium.png|1326| Changed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.click.chromium.png|2447| Changed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover day.chromium.png|0| Removed|

</details>


  <details><summary><b>DetailsHeader</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsHeader.Root.default.chromium.png|56720| Changed|
|DetailsHeader.Root.hoverFrozenFirst.chromium.png|56806| Changed|
|DetailsHeader.Root.hoverDraggable.chromium.png|56501| Changed|
|DetailsHeader.Root.hoverFrozenLast.chromium.png|56510| Changed|
|DetailsHeader.Root.borderWhileDragging.chromium.png|1163| Changed|
|DetailsHeader.Root.dropHint.chromium.png|1280| Changed|

</details>


  <details><summary><b>DetailsList</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsList.Checkbox Visible Always.click.chromium.png|207191| Changed|
|DetailsList.Checkbox Visible Always - RTL.default.chromium.png|207042| Changed|
|DetailsList.Checkbox Visible Always.hover.chromium.png|207480| Changed|
|DetailsList.Checkbox Visible Always - RTL.click.chromium.png|206325| Changed|
|DetailsList.Compact.click.chromium.png|199969| Changed|
|DetailsList.Checkbox Visible Always - RTL.hover.chromium.png|206787| Changed|
|DetailsList.Compact.hover.chromium.png|200180| Changed|
|DetailsList.Compact.default.chromium.png|200418| Changed|
|DetailsList.Checkbox Visible Always.default.chromium.png|207785| Changed|
|DetailsList.Grouped with Checkbox Hidden.default.chromium.png|242800| Changed|
|DetailsList.Grouped with Checkbox Hidden.click.chromium.png|242282| Changed|
|DetailsList.Grouped with Checkbox Hidden.hover.chromium.png|242527| Changed|
|DetailsList.Root.click.chromium.png|207054| Changed|
|DetailsList.Root.default.chromium.png|207630| Changed|
|DetailsList.Grouped.default.chromium.png|309233| Changed|
|DetailsList.Root.hover.chromium.png|207343| Changed|
|DetailsList.Grouped.click.chromium.png|308565| Changed|
|DetailsList.Single Selection Mode.default.chromium.png|203033| Changed|
|DetailsList.Single Selection Mode.click.chromium.png|202522| Changed|
|DetailsList.Grouped.hover.chromium.png|308892| Changed|
|DetailsList.Single Selection Mode.hover.chromium.png|202774| Changed|

</details>


  <details><summary><b>Dialog</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Blocking.default.chromium.png|68618| Changed|
|Dialog.Root - RTL.default.chromium.png|66175| Changed|
|Dialog.Large header.default.chromium.png|69956| Changed|
|Dialog.Root.default.chromium.png|68054| Changed|
|Dialog.Wide Dialog.default.chromium.png|65873| Changed|

</details>


  <details><summary><b>DocumentCard</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DocumentCard.With secondary title style.default.chromium.png|19235| Changed|
|DocumentCard.Not truncated.default.chromium.png|183798| Changed|
|DocumentCard.Compact with preview image.default.chromium.png|360779| Changed|
|DocumentCard.Compact with preview list.default.chromium.png|209938| Changed|

</details>


  <details><summary><b>Dropdown</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Custom Dropdown - RTL.default.chromium.png|5118| Changed|
|Dropdown.Custom Dropdown - RTL.hover.chromium.png|5171| Changed|
|Dropdown.Custom Dropdown.hover.chromium.png|5459| Changed|
|Dropdown.Custom Dropdown.default.chromium.png|5359| Changed|
|Dropdown.Disabled option selected.default.chromium.png|3843| Changed|
|Dropdown.Disabled option selected.hover.chromium.png|3842| Changed|
|Dropdown.Multiselect - RTL.default.chromium.png|5178| Changed|
|Dropdown.Multiselect - RTL.hover.chromium.png|5172| Changed|
|Dropdown.Multiselect.default.chromium.png|5178| Changed|
|Dropdown.Multiselect.hover.chromium.png|5172| Changed|
|Dropdown.Required without label.default.chromium.png|3632| Changed|
|Dropdown.Required without label.hover.chromium.png|3666| Changed|
|Dropdown.Required.default.chromium.png|7614| Changed|
|Dropdown.Required.hover.chromium.png|7601| Changed|
|Dropdown.Disabled option selected.click.chromium.png|1834054| Changed|
|Dropdown.Root - RTL.default.chromium.png|6041| Changed|
|Dropdown.Multiselect - RTL.hover item.chromium.png|1743283| Changed|
|Dropdown.Root - RTL.hover.chromium.png|6021| Changed|
|Dropdown.Custom Dropdown - RTL.click.chromium.png|1813571| Changed|
|Dropdown.Root.default.chromium.png|6108| Changed|
|Dropdown.Root.hover.chromium.png|6089| Changed|
|Dropdown.Disabled option selected.hover item.chromium.png|1833928| Changed|
|Dropdown.Multiselect - RTL.click.chromium.png|1743274| Changed|
|Dropdown.Required without label.click.chromium.png|1840816| Changed|
|Dropdown.Required.hover item.chromium.png|1839806| Changed|
|Dropdown.Multiselect.hover item.chromium.png|1747574| Changed|
|Dropdown.Multiselect.click.chromium.png|1747569| Changed|
|Dropdown.Custom Dropdown - RTL.hover item.chromium.png|1813460| Changed|
|Dropdown.Custom Dropdown.hover item.chromium.png|1814480| Changed|
|Dropdown.Required.click.chromium.png|1840063| Changed|
|Dropdown.Custom Dropdown.click.chromium.png|1814730| Changed|
|Dropdown.Required without label.hover item.chromium.png|1840673| Changed|
|Dropdown.Root - RTL.hover item.chromium.png|1810026| Changed|
|Dropdown.Root - RTL.click.chromium.png|1810164| Changed|
|Dropdown.Root.hover item.chromium.png|1810809| Changed|
|Dropdown.Root.click.chromium.png|1811051| Changed|

</details>


  <details><summary><b>Dropdown Disabled</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Disabled.Root.default.chromium.png|5921| Changed|
|Dropdown Disabled.Root.hover.chromium.png|5921| Changed|
|Dropdown Disabled.Root.click.chromium.png|971| Changed|

</details>


  <details><summary><b>Facepile</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Facepile.Custom button styles.default.chromium.png|65293| Changed|
|Facepile.Add face.default.chromium.png|64739| Changed|
|Facepile.Root.default.chromium.png|58420| Changed|
|Facepile.Root - RTL.default.chromium.png|57934| Changed|
|Facepile.Extra extra small.default.chromium.png|54590| Changed|
|Facepile.Overflow.default.chromium.png|53672| Changed|

</details>


  <details><summary><b>FocusTrapZones</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FocusTrapZones.Panel on its own.default.chromium.png|3339| Changed|
|FocusTrapZones.Panel on its own.click on panel close button.chromium.png|3405| Changed|
|FocusTrapZones.Dialog nested in Panel.click on panel close button.chromium.png|710829| Changed|
|FocusTrapZones.Dialog nested in Panel - RTL.default.chromium.png|710818| Changed|
|FocusTrapZones.Dialog nested in Panel - RTL.click on panel close button.chromium.png|710806| Changed|
|FocusTrapZones.Dialog nested in Panel.default.chromium.png|710840| Changed|

</details>


  <details><summary><b>FolderCover</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FolderCover.Large Default Cover.default.chromium.png|298| Changed|
|FolderCover.Shared Cover.default.chromium.png|233| Changed|
|FolderCover.Large Media Cover.default.chromium.png|507| Changed|
|FolderCover.Small Default Cover.default.chromium.png|168| Changed|
|FolderCover.Small Media Cover.default.chromium.png|212| Changed|

</details>


  <details><summary><b>Fonts</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Fonts.Chinese (Simplified).default.chromium.png|33673| Changed|
|Fonts.Armenian.default.chromium.png|54726| Changed|
|Fonts.Arabic.default.chromium.png|33771| Changed|
|Fonts.Chinese (Traditional).default.chromium.png|33673| Changed|
|Fonts.East European.default.chromium.png|33672| Changed|
|Fonts.Georgian.default.chromium.png|33716| Changed|
|Fonts.Weights.default.chromium.png|7701| Changed|
|Fonts.Hebrew.default.chromium.png|33734| Changed|
|Fonts.Greek.default.chromium.png|33606| Changed|
|Fonts.Cyrillic.default.chromium.png|33676| Changed|
|Fonts.West European.default.chromium.png|33673| Changed|
|Fonts.Vietnamese.default.chromium.png|33658| Changed|
|Fonts.Thai.default.chromium.png|33552| Changed|

</details>


  <details><summary><b>GroupedList</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|GroupedList.Root - RTL.click.chromium.png|47835| Changed|
|GroupedList.Root.click.chromium.png|46602| Changed|
|GroupedList.Root - RTL.default.chromium.png|49635| Changed|
|GroupedList.Root.default.chromium.png|49662| Changed|
|GroupedList.Root - RTL.hover.chromium.png|50216| Changed|
|GroupedList.Root.hover.chromium.png|50243| Changed|

</details>


  <details><summary><b>HoverCard</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|HoverCard.Root.default.chromium.png|954| Changed|
|HoverCard.Root - RTL.default.chromium.png|949| Changed|
|HoverCard.Root.fully expanded with test content.chromium.png|1813868| Changed|
|HoverCard.Root - RTL.fully expanded with test content.chromium.png|1812143| Changed|

</details>


  <details><summary><b>Icon</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Icon.Color.default.chromium.png|370| Changed|
|Icon.Root.default.chromium.png|441818| Changed|

</details>


  <details><summary><b>IconButton Scenarios (compat)</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|IconButton Scenarios (compat).normal icon button.hover icon.chromium.png|1116| Changed|
|IconButton Scenarios (compat).icon button with menu.hover icon.chromium.png|1933| Changed|
|IconButton Scenarios (compat).icon button with menu.icon.chromium.png|1927| Changed|
|IconButton Scenarios (compat).normal icon button.icon.chromium.png|1089| Changed|

</details>


  <details><summary><b>Image</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- CenterContain, image larger.default.chromium.png|19114| Changed|
|Image.Fit- centerCover, image larger.default.chromium.png|14587| Changed|
|Image.Fit- center, image larger.default.chromium.png|17040| Changed|
|Image.Fit- CenterContain, image taller.default.chromium.png|20413| Changed|
|Image.Fit- contain, image taller.default.chromium.png|23287| Changed|
|Image.Fit- center, image smaller.default.chromium.png|20163| Changed|
|Image.Fit- CenterContain, image wider.default.chromium.png|32299| Changed|
|Image.Fit- CenterContain, image smaller.default.chromium.png|34984| Changed|
|Image.Maximize frame, landscape container.default.chromium.png|9196| Changed|
|Image.Maximize frame, portrait container.default.chromium.png|43| Changed|
|Image.Fit- contain, image wider.default.chromium.png|45829| Changed|
|Image.Fit- cover, image taller.default.chromium.png|25343| Changed|
|Image.Fit- centerCover, image wider.default.chromium.png|34083| Changed|
|Image.Fit- centerCover, image taller.default.chromium.png|21752| Changed|
|Image.No fit, no w-h.default.chromium.png|12040| Changed|
|Image.Fit- none, image larger.default.chromium.png|19937| Changed|
|Image.Fit- none, image smaller.default.chromium.png|17170| Changed|
|Image.Fit- centerCover, image smaller.default.chromium.png|34984| Changed|
|Image.No fit, only height.default.chromium.png|85586| Changed|
|Image.Fit- cover, image wider.default.chromium.png|37302| Changed|
|Image.No fit, only width.default.chromium.png|96628| Changed|

</details>


  <details><summary><b>Keytip</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Disabled.default.chromium.png|135| Changed|
|Keytip.Offset.default.chromium.png|289| Changed|
|Keytip.Root.default.chromium.png|250| Changed|

</details>


  <details><summary><b>Label</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Disabled.default.chromium.png|2175| Changed|
|Label.Required.default.chromium.png|3072| Changed|
|Label.Root.default.chromium.png|1315| Changed|
|Label.Root - RTL.default.chromium.png|1318| Changed|

</details>


  <details><summary><b>Layer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Layer.Root.default.chromium.png|1842342| Changed|
|Layer.Root - RTL.default.chromium.png|1842130| Changed|

</details>


  <details><summary><b>Link</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link.Disabled.default.chromium.png|835| Changed|
|Link.Disabled.click.chromium.png|835| Changed|
|Link.Disabled.hover.chromium.png|835| Changed|
|Link.No Href Disabled.click.chromium.png|1376| Changed|
|Link.No Href.default.chromium.png|1769| Changed|
|Link.No Href Disabled.hover.chromium.png|1376| Changed|
|Link.Disabled.focus.chromium.png|835| Changed|
|Link.Root - RTL.default.chromium.png|536| Changed|
|Link.No Href.hover.chromium.png|2812| Changed|
|Link.No Href Disabled.focus.chromium.png|1376| Changed|
|Link.No Href Disabled.default.chromium.png|1376| Changed|
|Link.No Href.focus.chromium.png|2350| Changed|
|Link.Root - RTL.focus.chromium.png|812| Changed|
|Link.Root - RTL.hover.chromium.png|868| Changed|
|Link.Root.default.chromium.png|532| Changed|
|Link.Root - RTL.click.chromium.png|868| Changed|
|Link.Root.hover.chromium.png|863| Changed|
|Link.No Href.click.chromium.png|2812| Changed|
|Link.Root.focus.chromium.png|810| Changed|
|Link.Underlined Disabled.click.chromium.png|2230| Changed|
|Link.Root.click.chromium.png|863| Changed|
|Link.Underlined Disabled.hover.chromium.png|2230| Changed|
|Link.Underlined.click.chromium.png|3178| Changed|
|Link.Underlined Disabled.default.chromium.png|2230| Changed|
|Link.Underlined.hover.chromium.png|3178| Changed|
|Link.Underlined.focus.chromium.png|3170| Changed|
|Link.Underlined.default.chromium.png|1981| Changed|
|Link.Underlined Disabled.focus.chromium.png|2230| Changed|

</details>


  <details><summary><b>MediaTile</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MediaTile.Media tile with nameplate hidden until hover.hover.chromium.png|251| Changed|
|MediaTile.Media tile with two activity lines.default.chromium.png|446| Changed|
|MediaTile.Media tile with single activity line.default.chromium.png|251| Changed|
|MediaTile.Media tile with very long name and activity.hover.chromium.png|576| Changed|
|MediaTile.Media tile with single activity line.hover.chromium.png|251| Changed|
|MediaTile.Media tile with two activity lines.hover.chromium.png|446| Changed|
|MediaTile.Media tile with very long name and activity.default.chromium.png|576| Changed|

</details>


  <details><summary><b>MessageBar</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Blocked.default.chromium.png|2710| Changed|
|MessageBar.Error.default.chromium.png|2663| Changed|
|MessageBar.Root actions - RTL.default.chromium.png|4333| Changed|
|MessageBar.Root - RTL.default.chromium.png|2765| Changed|
|MessageBar.Root dismiss - RTL.default.chromium.png|10595| Changed|
|MessageBar.Root dismiss and action.default.chromium.png|8984| Changed|
|MessageBar.Root dismiss and action - RTL.default.chromium.png|8992| Changed|
|MessageBar.Root actions single line.default.chromium.png|18130| Changed|
|MessageBar.Root actions.default.chromium.png|4333| Changed|
|MessageBar.Root actions single line - RTL.default.chromium.png|17807| Changed|
|MessageBar.Root dismiss and action single line - RTL.default.chromium.png|24943| Changed|
|MessageBar.Root dismiss single line.default.chromium.png|10595| Changed|
|MessageBar.Severe Warning.default.chromium.png|3628| Changed|
|MessageBar.Root.default.chromium.png|2758| Changed|
|MessageBar.Root dismiss.default.chromium.png|10595| Changed|
|MessageBar.Root dismiss and action single line.default.chromium.png|25288| Changed|
|MessageBar.Success.default.chromium.png|2785| Changed|
|MessageBar.Warning.default.chromium.png|2712| Changed|
|MessageBar.Root overflow.default.chromium.png|47566| Changed|
|MessageBar.Root overflow - RTL.default.chromium.png|47502| Changed|
|MessageBar.Root multiline.default.chromium.png|28950| Changed|
|MessageBar.Root truncated - RTL.default.chromium.png|121440| Changed|
|MessageBar.Root truncated.default.chromium.png|121441| Changed|
|MessageBar.Root multiline - RTL.default.chromium.png|28972| Changed|

</details>


  <details><summary><b>Modal</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Modal.Modeless.default.chromium.png|1837999| Changed|
|Modal.Root.default.chromium.png|1124147| Changed|

</details>


  <details><summary><b>Nav</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Nav.Disabled - RTL.click.chromium.png|15933| Changed|
|Nav.Disabled - RTL.default.chromium.png|27033| Changed|
|Nav.Disabled - RTL.hover.chromium.png|27033| Changed|
|Nav.Disabled.default.chromium.png|27032| Changed|
|Nav.Disabled.hover.chromium.png|27032| Changed|
|Nav.Disabled.click.chromium.png|15937| Changed|
|Nav.Root - RTL.default.chromium.png|30715| Changed|
|Nav.Root - RTL.click.chromium.png|23509| Changed|
|Nav.Root - RTL.hover.chromium.png|31107| Changed|
|Nav.Root.hover.chromium.png|31118| Changed|
|Nav.Root.click.chromium.png|23513| Changed|
|Nav.Root.default.chromium.png|30725| Changed|

</details>


  <details><summary><b>OverflowSet</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Root - RTL.default.chromium.png|7709| Changed|
|OverflowSet.Root.default.chromium.png|7728| Changed|
|OverflowSet.Root.default.chromium_1.png|2081| Changed|
|OverflowSet.Root - RTL.default.chromium_1.png|2082| Changed|

</details>


  <details><summary><b>OverflowSet variant</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet variant.Vertical Direction.default.chromium.png|1795| Changed|

</details>


  <details><summary><b>Overlay</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Overlay.Root - RTL.default.chromium.png|1841846| Changed|
|Overlay.Root.default.chromium.png|1842192| Changed|
|Overlay.Dark.default.chromium.png|1105854| Changed|

</details>


  <details><summary><b>Panel</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Panel.Custom.default.chromium.png|1080| Changed|
|Panel.Medium right - RTL.default.chromium.png|3210| Changed|
|Panel.Custom anchored left.default.chromium.png|3228| Changed|
|Panel.Extra large right.default.chromium.png|3320| Changed|
|Panel.Large fixed right.default.chromium.png|3331| Changed|
|Panel.Large right.default.chromium.png|3033| Changed|
|Panel.Medium right.default.chromium.png|3210| Changed|
|Panel.SearchBox and Right Panel - RTL.default.chromium.png|950| Changed|
|Panel.SearchBox and Right Panel - RTL.click.chromium.png|1512| Changed|
|Panel.SearchBox and Right Panel.default.chromium.png|938| Changed|
|Panel.SearchBox and Right Panel.click.chromium.png|1510| Changed|
|Panel.Small fixed right w- close button.default.chromium.png|3340| Changed|
|Panel.Small fixed right w- close button - RTL.default.chromium.png|3341| Changed|
|Panel.Small left w- close button - RTL.default.chromium.png|2981| Changed|
|Panel.Small left w- close button.default.chromium.png|2977| Changed|
|Panel.Small fluid right.default.chromium.png|1242| Changed|
|Panel.With no header, close button.default.chromium.png|2711| Changed|
|Panel.With footer at the bottom.default.chromium.png|5774| Changed|
|Panel.With no navigation.default.chromium.png|3294| Changed|
|Panel.With custom navigation.default.chromium.png|4184| Changed|

</details>


  <details><summary><b>PeoplePicker</b> 29 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Compact selected - RTL.default.chromium.png|27655| Changed|
|PeoplePicker.Compact selected.default.chromium.png|28285| Changed|
|PeoplePicker.Compact.default.chromium.png|2805| Changed|
|PeoplePicker.List selected - RTL.default.chromium.png|16338| Changed|
|PeoplePicker.List selected.default.chromium.png|16414| Changed|
|PeoplePicker.List.default.chromium.png|2805| Changed|
|PeoplePicker.Normal Overflow selected - RTL.default.chromium.png|79387| Changed|
|PeoplePicker.Normal Overflow selected.default.chromium.png|88128| Changed|
|PeoplePicker.Normal selected - RTL.default.chromium.png|27655| Changed|
|PeoplePicker.Normal selected.default.chromium.png|28285| Changed|
|PeoplePicker.Compact selected - RTL.suggestions.chromium.png|57546| Changed|
|PeoplePicker.Compact disabled.chromium.png|144| Changed|
|PeoplePicker.Normal with text.default.chromium.png|2805| Changed|
|PeoplePicker.Normal.default.chromium.png|2805| Changed|
|PeoplePicker.List.suggestions.chromium.png|52734| Changed|
|PeoplePicker.List disabled.chromium.png|269| Changed|
|PeoplePicker.Compact.suggestions.chromium.png|52738| Changed|
|PeoplePicker.Compact selected.suggestions.chromium.png|57148| Changed|
|PeoplePicker.List selected.suggestions.chromium.png|52443| Changed|
|PeoplePicker.List selected - RTL.suggestions.chromium.png|52413| Changed|
|PeoplePicker.Normal disabled with placeholder.chromium.png|279| Changed|
|PeoplePicker.Normal disabled.chromium.png|144| Changed|
|PeoplePicker.Normal Overflow selected - RTL.suggestions.chromium.png|53161| Changed|
|PeoplePicker.Normal with placeholder.chromium.png|464| Changed|
|PeoplePicker.Normal selected.suggestions.chromium.png|57336| Changed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|52561| Changed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|57723| Changed|
|PeoplePicker.Normal.suggestions.chromium.png|52734| Changed|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|53287| Changed|

</details>


  <details><summary><b>Persona</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona.Initials - RTL.default.chromium.png|9005| Changed|
|Persona.Initials.default.chromium.png|9837| Changed|
|Persona.Persona with children.default.chromium.png|15947| Changed|
|Persona.default size, details hidden.default.chromium.png|1103| Changed|
|Persona.size10 (tiny).default.chromium.png|15279| Changed|
|Persona.size100 (extraLarge) - RTL.default.chromium.png|49445| Changed|
|Persona.size100 (extraLarge).default.chromium.png|53928| Changed|
|Persona.size120 - RTL.default.chromium.png|31760| Changed|
|Persona.size32.default.chromium.png|9965| Changed|
|Persona.size28 (extraSmall).default.chromium.png|18767| Changed|
|Persona.size24 (extraExtraSmall).default.chromium.png|17740| Changed|
|Persona.size120.default.chromium.png|34754| Changed|
|Persona.size40 (small).default.chromium.png|22779| Changed|
|Persona.default size, presences.default.chromium.png|116554| Changed|
|Persona.size48 (regular).default.chromium.png|29250| Changed|
|Persona.size72 (large).default.chromium.png|44625| Changed|

</details>


  <details><summary><b>Pivot</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot.Count and icon - RTL.hover-selected.chromium.png|30329| Changed|
|Pivot.Count and icon - RTL.default.chromium.png|30519| Changed|
|Pivot.Count and icon - RTL.click.chromium.png|30815| Changed|
|Pivot.Count and icon.hover-selected.chromium.png|31210| Changed|
|Pivot.Count and icon.default.chromium.png|31097| Changed|
|Pivot.Count and icon - RTL.hover.chromium.png|30583| Changed|
|Pivot.Count and icon.click.chromium.png|31146| Changed|
|Pivot.Root.default.chromium.png|12196| Changed|
|Pivot.Root.click.chromium.png|12307| Changed|
|Pivot.Large.hover.chromium.png|18769| Changed|
|Pivot.Large.hover-selected.chromium.png|18782| Changed|
|Pivot.Count and icon.hover.chromium.png|31162| Changed|
|Pivot.Tabs - RTL.click.chromium.png|8038| Changed|
|Pivot.Large.click.chromium.png|18811| Changed|
|Pivot.Tabs large.click.chromium.png|11023| Changed|
|Pivot.Root.hover-selected.chromium.png|12272| Changed|
|Pivot.Tabs - RTL.default.chromium.png|7639| Changed|
|Pivot.Root.hover.chromium.png|12260| Changed|
|Pivot.Tabs - RTL.hover-selected.chromium.png|7649| Changed|
|Pivot.Tabs.click.chromium.png|8143| Changed|
|Pivot.Tabs large.default.chromium.png|10997| Changed|
|Pivot.Large.default.chromium.png|18704| Changed|
|Pivot.Tabs large.hover.chromium.png|11803| Changed|
|Pivot.Tabs - RTL.hover.chromium.png|8359| Changed|
|Pivot.Tabs large.hover-selected.chromium.png|10995| Changed|
|Pivot.Tabs.default.chromium.png|8137| Changed|
|Pivot.Tabs.hover-selected.chromium.png|8133| Changed|
|Pivot.Tabs.hover.chromium.png|8879| Changed|

</details>


  <details><summary><b>Pivot - Overflow</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|9018| Changed|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|8228| Changed|
|Pivot - Overflow.Root.Medium.chromium.png|4242| Changed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|11935| Changed|
|Pivot - Overflow.Tabs - RTL.Medium.chromium.png|4917| Changed|
|Pivot - Overflow.Root.Wide.chromium.png|21358| Changed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|10308| Changed|
|Pivot - Overflow.Tabs - RTL.Wide.chromium.png|19437| Changed|
|Pivot - Overflow.Root.chromium.png|3717| Changed|
|Pivot - Overflow.Tabs - RTL.chromium.png|5006| Changed|

</details>


  <details><summary><b>ProgressIndicator</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressIndicator.50% - RTL.default.chromium.png|4955| Changed|
|ProgressIndicator.0%.default.chromium.png|4995| Changed|
|ProgressIndicator.100%.default.chromium.png|5104| Changed|
|ProgressIndicator.50%.default.chromium.png|5049| Changed|

</details>


  <details><summary><b>Rating</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Rating.Disabled.click.chromium.png|1723| Changed|
|Rating.Disabled.default.chromium.png|1723| Changed|
|Rating.Large.click.chromium.png|3188| Changed|
|Rating.Rated - RTL.default.chromium.png|2937| Changed|
|Rating.Rated - RTL.click.chromium.png|2952| Changed|
|Rating.Large.default.chromium.png|3125| Changed|
|Rating.Root.click.chromium.png|2948| Changed|
|Rating.Rated.default.chromium.png|2934| Changed|
|Rating.Root.default.chromium.png|2900| Changed|
|Rating.Rated.click.chromium.png|2948| Changed|

</details>


  <details><summary><b>ResizeGroup</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ResizeGroup.Root - RTL.default.chromium.png|41217| Changed|
|ResizeGroup.Root.default.chromium.png|41355| Changed|
|ResizeGroup.Root.click overflow.chromium.png|5303| Changed|
|ResizeGroup.Root - RTL.click overflow.chromium.png|5304| Changed|

</details>


  <details><summary><b>ScrollablePane</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane.Default ScrollablePane Example.scrolled.chromium.png|64939| Changed|
|ScrollablePane.Default ScrollablePane Example.default.chromium.png|45389| Changed|

</details>


  <details><summary><b>ScrollablePane Details List</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll down to the bottom.chromium.png|13140| Changed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.default.chromium.png|12908| Changed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll down by a small amount so that the first row is still visible.chromium.png|12747| Changed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll up to the top.chromium.png|12908| Changed|

</details>


  <details><summary><b>ScrollablePane Grouped Details List</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.If groups are expanded, when header becomes sticky, horizontal scrollbar should be visible.chromium.png|2608815| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.On expanding a group, horizontal scrollbar should be visible.chromium.png|2536295| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.default- scrollbars should be visible.chromium.png|2499490| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.If groups are expanded, on horizontal scroll, over scroll should not happen for content container.chromium.png|2606876| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility when header is sticky and scrollLeft is non-zero.chromium.png|2499347| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling up to the top with non-zero scrollLeft.chromium.png|2499199| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling down to the bottom.chromium.png|2499810| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility when header is sticky.chromium.png|2499522| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibilty after scrolling left.chromium.png|2499199| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling up to the top.chromium.png|2499490| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling down to the bottom with non-zero scrollLeft.chromium.png|2499626| Changed|

</details>


  <details><summary><b>SearchBox</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.Full - RTL.default.chromium.png|3682| Changed|
|SearchBox.Full - RTL.default.chromium_1.png|3036| Changed|
|SearchBox.Full.default.chromium.png|3708| Changed|
|SearchBox.Full.default.chromium_1.png|3023| Changed|
|SearchBox.Root - RTL.default.chromium.png|3112| Changed|
|SearchBox.Root.default.chromium.png|3101| Changed|
|SearchBox.Root - RTL.default.chromium_1.png|3949| Changed|
|SearchBox.Root.default.chromium_1.png|3948| Changed|
|SearchBox.ShowIcon - RTL.default.chromium_1.png|10114| Changed|
|SearchBox.ShowIcon.default.chromium.png|9583| Changed|
|SearchBox.ShowIcon.default.chromium_1.png|10211| Changed|
|SearchBox.ShowIcon - RTL.default.chromium.png|9479| Changed|

</details>


  <details><summary><b>Separator</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Separator.Horizontal End Aligned - RTL.default.chromium.png|17550| Changed|
|Separator.Horizontal End Aligned.default.chromium.png|17422| Changed|
|Separator.Horizontal Start Aligned.default.chromium.png|17550| Changed|
|Separator.Horizontal Start Aligned - RTL.default.chromium.png|17422| Changed|
|Separator.Root - RTL.default.chromium.png|17422| Changed|
|Separator.Root.default.chromium.png|17422| Changed|
|Separator.Vertical Center Aligned - RTL.default.chromium.png|24112| Changed|
|Separator.Vertical End Aligned - RTL.default.chromium.png|24112| Changed|
|Separator.Vertical End Aligned.default.chromium.png|24111| Changed|
|Separator.Vertical Center Aligned.default.chromium.png|24111| Changed|
|Separator.Vertical Start Aligned - RTL.default.chromium.png|24269| Changed|
|Separator.Vertical Start Aligned.default.chromium.png|24269| Changed|

</details>


  <details><summary><b>Shimmer</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shimmer.Circle Gap Line - RTL.default.chromium.png|522| Changed|
|Shimmer.Basic.default.chromium.png|403| Changed|
|Shimmer.Custom elements on themed background.default.chromium.png|23845| Changed|
|Shimmer.50% width.default.chromium.png|203| Changed|
|Shimmer.Circle Gap Line.default.chromium.png|522| Changed|
|Shimmer.Custom elements.default.chromium.png|618| Changed|
|Shimmer.Data not loaded.default.chromium.png|1036| Changed|
|Shimmer.Data loaded.default.chromium.png|398| Changed|

</details>


  <details><summary><b>Signals</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Signals.Awaiting approval.default.chromium.png|758| Changed|
|Signals.Blocked.default.chromium.png|457| Changed|
|Signals.Comments (count).default.chromium.png|665| Changed|
|Signals.Comments.default.chromium.png|634| Changed|
|Signals.Emailed.default.chromium.png|447| Changed|
|Signals.Follow.default.chromium.png|644| Changed|
|Signals.Malware detected.default.chromium.png|898| Changed|
|Signals.Mention.default.chromium.png|619| Changed|
|Signals.Missing metadata.default.chromium.png|849| Changed|
|Signals.New (positioning).default.chromium.png|96| Changed|
|Signals.New.default.chromium.png|284| Changed|
|Signals.NotFollow.default.chromium.png|805| Changed|
|Signals.Read-only.default.chromium.png|616| Changed|
|Signals.Record.default.chromium.png|493| Changed|
|Signals.Shared.default.chromium.png|564| Changed|
|Signals.Someone checked out.default.chromium.png|983| Changed|
|Signals.Trending.default.chromium.png|506| Changed|
|Signals.Unseen edit.default.chromium.png|725| Changed|
|Signals.Unseen reply.default.chromium.png|687| Changed|
|Signals.Warning.default.chromium.png|637| Changed|
|Signals.You checked out.default.chromium.png|754| Changed|

</details>


  <details><summary><b>Slider</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Disabled.default.chromium.png|4036| Changed|
|Slider.EqualMinMax.default.chromium.png|4337| Changed|
|Slider.Max not multiple of step.default.chromium.png|4714| Changed|
|Slider.Root - RTL.default.chromium.png|4472| Changed|
|Slider.Root.default.chromium.png|4485| Changed|
|Slider.Step less than 1.default.chromium.png|4431| Changed|
|Slider.Vertical.default.chromium.png|806| Changed|
|Slider.Disabled.hover.chromium.png|0| Removed|
|Slider.EqualMinMax.hover.chromium.png|0| Removed|
|Slider.Max not multiple of step.hover.chromium.png|0| Removed|
|Slider.Root - RTL.hover.chromium.png|0| Removed|
|Slider.Root.hover.chromium.png|0| Removed|
|Slider.Step less than 1.hover.chromium.png|0| Removed|
|Slider.Vertical.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>SpinButton</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton.Disabled.click input.chromium.png|3367| Changed|
|SpinButton.Disabled.default.chromium.png|3367| Changed|
|SpinButton.Disabled.hover arrow.chromium.png|3367| Changed|
|SpinButton.Disabled.hover input.chromium.png|3367| Changed|
|SpinButton.Disabled.mouseDown arrow.chromium.png|3367| Changed|
|SpinButton.Label at end - RTL.default.chromium.png|3843| Changed|
|SpinButton.Label at end with icon - RTL.default.chromium.png|3877| Changed|
|SpinButton.Label at end with icon.default.chromium.png|3860| Changed|
|SpinButton.Label at end.default.chromium.png|3839| Changed|
|SpinButton.Label on bottom with icon.default.chromium.png|8067| Changed|
|SpinButton.Label on bottom.default.chromium.png|8020| Changed|
|SpinButton.Label on top with icon.default.chromium.png|18357| Changed|
|SpinButton.Root.click input.chromium.png|4501| Changed|
|SpinButton.Root.default.chromium.png|3786| Changed|
|SpinButton.Root.hover arrow.chromium.png|4540| Changed|
|SpinButton.Root.hover input.chromium.png|3910| Changed|
|SpinButton.Root.mouseDown arrow.chromium.png|3859| Changed|
|SpinButton.With icon - RTL.click input.chromium.png|4467| Changed|
|SpinButton.With icon - RTL.default.chromium.png|3799| Changed|
|SpinButton.With icon - RTL.hover arrow.chromium.png|4503| Changed|
|SpinButton.Label on top.default.chromium.png|16150| Changed|
|SpinButton.With icon - RTL.hover input.chromium.png|3913| Changed|
|SpinButton.With icon.click input.chromium.png|4458| Changed|
|SpinButton.With icon.default.chromium.png|3782| Changed|
|SpinButton.With icon - RTL.mouseDown arrow.chromium.png|3862| Changed|
|SpinButton.With icon.hover arrow.chromium.png|4497| Changed|
|SpinButton.With icon.mouseDown arrow.chromium.png|3845| Changed|
|SpinButton.With icon.hover input.chromium.png|3895| Changed|

</details>


  <details><summary><b>Spinner</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Spinner.Extra small.default.chromium.png|626| Changed|
|Spinner.Label on the left.default.chromium.png|1202| Changed|
|Spinner.Label on the right.default.chromium.png|1233| Changed|
|Spinner.Label.default.chromium.png|2987| Changed|
|Spinner.Large.default.chromium.png|1884| Changed|
|Spinner.Medium.default.chromium.png|1191| Changed|
|Spinner.Label at top.default.chromium.png|3141| Changed|
|Spinner.Small.default.chromium.png|892| Changed|

</details>


  <details><summary><b>Stack</b> 44 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Stack.Horizontal Stack - Box shadow around items.default.chromium.png|65200| Changed|
|Stack.Horizontal Stack - Default - RTL.default.chromium.png|64953| Changed|
|Stack.Horizontal Stack - Default.default.chromium.png|64953| Changed|
|Stack.Horizontal Stack - Horizontally centered.default.chromium.png|64990| Changed|
|Stack.Horizontal Stack - Gap.default.chromium.png|64953| Changed|
|Stack.Horizontal Stack - Gap - RTL.default.chromium.png|64953| Changed|
|Stack.Horizontal Stack - Growing items.default.chromium.png|80997| Changed|
|Stack.Horizontal Stack - Reversed - RTL.default.chromium.png|64990| Changed|
|Stack.Horizontal Stack - Padding.default.chromium.png|88569| Changed|
|Stack.Horizontal Stack - Reversed.default.chromium.png|64990| Changed|
|Stack.Horizontal Stack - Item alignments - RTL.default.chromium.png|314662| Changed|
|Stack.Horizontal Stack - Right-aligned - RTL.default.chromium.png|64990| Changed|
|Stack.Horizontal Stack - Right-aligned.default.chromium.png|64990| Changed|
|Stack.Horizontal Stack - Bottom-aligned.default.chromium.png|367037| Changed|
|Stack.Horizontal Stack - Space evenly.default.chromium.png|64990| Changed|
|Stack.Horizontal Stack - Shrinking items.default.chromium.png|217322| Changed|
|Stack.Horizontal Stack - Space between.default.chromium.png|64953| Changed|
|Stack.Horizontal Stack - Space around.default.chromium.png|64990| Changed|
|Stack.Horizontal Stack - Wrap with specified vertical gap.default.chromium.png|66199| Changed|
|Stack.Horizontal Stack - Item alignments.default.chromium.png|314630| Changed|
|Stack.Horizontal Stack - Wrap.default.chromium.png|66199| Changed|
|Stack.Horizontal Stack - Wrap - RTL.default.chromium.png|66220| Changed|
|Stack.Vertical Stack - Default - RTL.default.chromium.png|185191| Changed|
|Stack.Vertical Stack - Horizontally centered.default.chromium.png|185228| Changed|
|Stack.Vertical Stack - Default.default.chromium.png|185191| Changed|
|Stack.Vertical Stack - Right-aligned.default.chromium.png|185228| Changed|
|Stack.Vertical Stack - Gap.default.chromium.png|209530| Changed|
|Stack.Vertical Stack - Right-aligned - RTL.default.chromium.png|185228| Changed|
|Stack.Vertical Stack - Reversed.default.chromium.png|185187| Changed|
|Stack.Vertical Stack - Bottom-aligned.default.chromium.png|367037| Changed|
|Stack.Horizontal Stack - Vertically centered.default.chromium.png|367037| Changed|
|Stack.Vertical Stack - Reversed - RTL.default.chromium.png|185187| Changed|
|Stack.Vertical Stack - Item alignments - RTL.default.chromium.png|254257| Changed|
|Stack.Vertical Stack - Padding.default.chromium.png|209305| Changed|
|Stack.Vertical Stack - Item alignments.default.chromium.png|254224| Changed|
|Stack.Vertical Stack - Growing items.default.chromium.png|273403| Changed|
|Stack.Vertical Stack - Space evenly.default.chromium.png|367037| Changed|
|Stack.Vertical Stack - Box shadow around items.default.chromium.png|424953| Changed|
|Stack.Vertical Stack - Vertically centered.default.chromium.png|367037| Changed|
|Stack.Vertical Stack - Wrap - RTL.default.chromium.png|361346| Changed|
|Stack.Vertical Stack - Wrap.default.chromium.png|361338| Changed|
|Stack.Vertical Stack - Shrinking items.default.chromium.png|289876| Changed|
|Stack.Vertical Stack - Space between.default.chromium.png|367730| Changed|
|Stack.Vertical Stack - Space around.default.chromium.png|367116| Changed|

</details>


  <details><summary><b>Sticky breadcrumb and sticky details list header</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll back to the top when horizontal scroll is non-zero.chromium.png|16086| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll up to the top.chromium.png|16248| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.default.chromium.png|16248| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down by a small amount so that the first row is still visible and the header becomes sticky.chromium.png|16198| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down to the bottom.chromium.png|16821| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down when horizontal scroll is non-zero.chromium.png|16683| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll right.chromium.png|16086| Changed|

</details>


  <details><summary><b>SwatchColorPicker</b> 42 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchColorPicker.Circle - RTL.Focused.chromium.png|8348| Changed|
|SwatchColorPicker.Circle - RTL.default.chromium.png|8266| Changed|
|SwatchColorPicker.Circle over 24px size.Focused.chromium.png|10350| Changed|
|SwatchColorPicker.Circle over 24px size.default.chromium.png|10291| Changed|
|SwatchColorPicker.Circle.Focused.chromium.png|8814| Changed|
|SwatchColorPicker.Disabled.Focused.chromium.png|8511| Changed|
|SwatchColorPicker.Circle.default.chromium.png|8739| Changed|
|SwatchColorPicker.Disabled.default.chromium.png|8511| Changed|
|SwatchColorPicker.Multiple rows.default.chromium.png|12530| Changed|
|SwatchColorPicker.Multiple rows.Focused.chromium.png|12605| Changed|
|SwatchColorPicker.Square over 24px size.Focused.chromium.png|9870| Changed|
|SwatchColorPicker.Square over 24px size.default.chromium.png|9817| Changed|
|SwatchColorPicker.Square.Focused.chromium.png|8879| Changed|
|SwatchColorPicker.Square.default.chromium.png|8815| Changed|
|SwatchColorPicker.Circle - RTL.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle - RTL.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Circle - RTL.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle - RTL.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle over 24px size.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Circle.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Disabled.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Multiple rows.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Square over 24px size.Selected and Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Square.Hovered.chromium.png|0| Removed|
|SwatchColorPicker.Square.Mousedown.chromium.png|0| Removed|
|SwatchColorPicker.Square.Selected and Focused.chromium.png|0| Removed|
|SwatchColorPicker.Square.Selected and Hovered.chromium.png|0| Removed|

</details>


  <details><summary><b>TagItem</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagItem.Selected - RTL.Tag Item Clear Button Selected Hover.chromium.png|6623| Changed|
|TagItem.Selected - RTL.Tag Item Hover.chromium.png|7452| Changed|
|TagItem.Selected - RTL.default.chromium.png|7480| Changed|
|TagItem.Selected.Tag Item Clear Button Hover.chromium.png|7884| Changed|
|TagItem.Selected - RTL.Tag Item Clear Button Hover.chromium.png|7436| Changed|
|TagItem.Selected - RTL.Tag Item Select.chromium.png|6673| Changed|
|TagItem.Selected.default.chromium.png|7774| Changed|
|TagItem.Selected.Tag Item Clear Button Selected Hover.chromium.png|8673| Changed|
|TagItem.Selected.Tag Item Hover.chromium.png|7848| Changed|
|TagItem.Selected.Tag Item Select.chromium.png|8624| Changed|

</details>


  <details><summary><b>TagPicker</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Root.Open Suggestion Menu.chromium.png|3088| Changed|
|TagPicker.Root.Suggestion Menu Item Hover.chromium.png|3088| Changed|
|TagPicker.Root.default.chromium.png|2580| Changed|
|TagPicker.Selected - RTL.Open Suggestion Menu.chromium.png|7645| Changed|
|TagPicker.Selected - RTL.Suggestion Menu Item Hover.chromium.png|7645| Changed|
|TagPicker.Selected - RTL.default.chromium.png|7480| Changed|
|TagPicker.Selected.Open Suggestion Menu.chromium.png|7910| Changed|
|TagPicker.Selected.default.chromium.png|7774| Changed|
|TagPicker.With long tag.default.chromium.png|1517| Changed|
|TagPicker.Selected.Suggestion Menu Item Hover.chromium.png|7910| Changed|
|TagPicker.TagPicker disabled.chromium.png|137| Changed|

</details>


  <details><summary><b>TeachingBubble</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TeachingBubble.SmallHeadline.default.chromium.png|85039| Changed|
|TeachingBubble.WideIllustration.default.chromium.png|230988| Changed|
|TeachingBubble.WideIllustration - RTL.default.chromium.png|230297| Changed|

</details>


  <details><summary><b>Text</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.Block.default.chromium.png|21270| Changed|
|Text.Root - RTL.default.chromium.png|682| Changed|
|Text.Root.default.chromium.png|676| Changed|
|Text.No wrap.default.chromium.png|39300| Changed|
|Text.Wrap.default.chromium.png|13556| Changed|

</details>


  <details><summary><b>TextField</b> 60 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextField.Borderless.click.chromium.png|8219| Changed|
|TextField.Borderless.default.chromium.png|7767| Changed|
|TextField.Borderless.hover.chromium.png|7821| Changed|
|TextField.Error - RTL.hover.chromium.png|3472| Changed|
|TextField.Error - RTL.click.chromium.png|4334| Changed|
|TextField.Error - RTL.default.chromium.png|3417| Changed|
|TextField.Error.click.chromium.png|4329| Changed|
|TextField.Disabled.default.chromium.png|7335| Changed|
|TextField.Disabled.hover.chromium.png|7335| Changed|
|TextField.Error.hover.chromium.png|3468| Changed|
|TextField.Icon.click.chromium.png|8155| Changed|
|TextField.Error.default.chromium.png|3414| Changed|
|TextField.Disabled.click.chromium.png|7335| Changed|
|TextField.Icon.default.chromium.png|7740| Changed|
|TextField.Icon.hover.chromium.png|7790| Changed|
|TextField.Icon - RTL.default.chromium.png|7719| Changed|
|TextField.Multiline - RTL.click.chromium.png|8465| Changed|
|TextField.Icon - RTL.click.chromium.png|8131| Changed|
|TextField.Placeholder - RTL.default.chromium.png|7607| Changed|
|TextField.Icon - RTL.hover.chromium.png|7769| Changed|
|TextField.Multiline nonresizable.default.chromium.png|7911| Changed|
|TextField.Multiline nonresizable.hover.chromium.png|7983| Changed|
|TextField.Multiline.click.chromium.png|8489| Changed|
|TextField.Multiline.default.chromium.png|7905| Changed|
|TextField.Placeholder - RTL.click.chromium.png|8036| Changed|
|TextField.Placeholder.click.chromium.png|8036| Changed|
|TextField.Multiline nonresizable.click.chromium.png|8495| Changed|
|TextField.Placeholder - RTL.hover.chromium.png|7661| Changed|
|TextField.Multiline - RTL.default.chromium.png|7881| Changed|
|TextField.Placeholder.hover.chromium.png|7662| Changed|
|TextField.Multiline - RTL.hover.chromium.png|7953| Changed|
|TextField.Prefix with Value, Disabled - RTL.hover.chromium.png|2104| Changed|
|TextField.Multiline.hover.chromium.png|7977| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.hover.chromium.png|2437| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.default.chromium.png|2428| Changed|
|TextField.Prefix with Value, Disabled.hover.chromium.png|2094| Changed|
|TextField.Required.hover.chromium.png|7571| Changed|
|TextField.Root.default.chromium.png|7475| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.hover.chromium.png|2428| Changed|
|TextField.Prefix with Value, Disabled.click.chromium.png|2094| Changed|
|TextField.Prefix with Value, Disabled - RTL.click.chromium.png|2104| Changed|
|TextField.Placeholder.default.chromium.png|7608| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.click.chromium.png|2437| Changed|
|TextField.Root.click.chromium.png|7923| Changed|
|TextField.Required.click.chromium.png|7968| Changed|
|TextField.Prefix with Value, Disabled - RTL.default.chromium.png|2104| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.click.chromium.png|2428| Changed|
|TextField.Suffix - RTL.default.chromium.png|2256| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.default.chromium.png|2437| Changed|
|TextField.Suffix - RTL.hover.chromium.png|2425| Changed|
|TextField.Underlined.hover.chromium.png|2786| Changed|
|TextField.Suffix.click.chromium.png|3113| Changed|
|TextField.Suffix.default.chromium.png|2253| Changed|
|TextField.Underlined.default.chromium.png|2679| Changed|
|TextField.Prefix with Value, Disabled.default.chromium.png|2094| Changed|
|TextField.Suffix.hover.chromium.png|2421| Changed|
|TextField.Root.hover.chromium.png|7529| Changed|
|TextField.Underlined.click.chromium.png|3310| Changed|
|TextField.Required.default.chromium.png|7519| Changed|
|TextField.Suffix - RTL.click.chromium.png|3117| Changed|

</details>


  <details><summary><b>ThemeProvider</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider.Customized styles.default.chromium.png|5721| Changed|
|ThemeProvider.Customized theme.default.chromium.png|5831| Changed|
|ThemeProvider.Default theme.default.chromium.png|4107| Changed|
|ThemeProvider.Customized theme - nested ThemeProvider.default.chromium.png|9673| Changed|

</details>


  <details><summary><b>ThemeProvider with Customizer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider with Customizer.ThemeProvider wraps Customizer.default.chromium.png|10895| Changed|
|ThemeProvider with Customizer.Customizer wraps ThemeProvider.default.chromium.png|9330| Changed|

</details>


  <details><summary><b>ThemeProvider with loadTheme</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ThemeProvider with loadTheme.Use updated global theme.default.chromium.png|5916| Changed|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.default.chromium.png|16154| Changed|
|ThemeProvider with loadTheme.Use contextual theme over global theme if defined.global theme changed.chromium.png|19411| Changed|
|ThemeProvider with loadTheme.Use updated global theme.global theme changed.chromium.png|7110| Changed|

</details>


  <details><summary><b>Tile</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tile.Document tile with fit landscape image.default.chromium.png|437| Changed|
|Tile.Document tile with fit portrait image.default.chromium.png|437| Changed|
|Tile.Tile with no content and long text.default.chromium.png|691| Changed|
|Tile.Document tile with icon-sized image.default.chromium.png|437| Changed|
|Tile.Document tile with icon.default.chromium.png|328| Changed|

</details>


  <details><summary><b>Toggle</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toggle.Disabled checked.default.chromium.png|2756| Changed|
|Toggle.Checked - RTL.default.chromium.png|3031| Changed|
|Toggle.Unchecked - RTL.default.chromium.png|2909| Changed|
|Toggle.Checked.default.chromium.png|3009| Changed|
|Toggle.Disabled unchecked.default.chromium.png|2723| Changed|
|Toggle.Unchecked.default.chromium.png|2888| Changed|
|Toggle.With inline label and without onText and offText.default.chromium.png|3483| Changed|
|Toggle.With inline label (JSX Element) and without onText and offText.default.chromium.png|5086| Changed|
|Toggle.With inline label disabled and without onText and offText.default.chromium.png|3325| Changed|
|Toggle.With inline label disabled.default.chromium.png|5229| Changed|
|Toggle.With inline label.default.chromium.png|5071| Changed|
|Toggle.With inline label (JSX Element).default.chromium.png|7617| Changed|

</details>


  <details><summary><b>Tooltip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Default.default.chromium.png|506| Changed|

</details>


  <details><summary><b>Tooltip - Multiple</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip - Multiple.Two Tooltips.hover outer.chromium.png|615| Changed|
|Tooltip - Multiple.Two Tooltips.hover inner.chromium.png|827| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>